#! /bin/sh /usr/share/dpatch/dpatch-run

## 10_debian-fhs-defaults by Thomas Schmidt <thomas.schmidt@in.stud.tu-ilmenau.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change several default-paths for Debian FHS conformance

@DPATCH@
diff -urNad vdr-plugin-mp3-0.10.2~/data-mp3.c vdr-plugin-mp3-0.10.2/data-mp3.c
--- vdr-plugin-mp3-0.10.2~/data-mp3.c	2009-11-29 04:05:21.000000000 +0100
+++ vdr-plugin-mp3-0.10.2/data-mp3.c	2010-01-23 20:37:21.000000000 +0100
@@ -36,8 +36,8 @@
 #define di(x) ; 
 #endif
 
-const char *imagecache = "/var/cache/images/mp3";
-const char *imageconv  = "image_convert.sh";
+const char *imagecache = "/var/cache/vdr-plugin-mp3/images";
+const char *imageconv  = "/usr/share/vdr-plugin-mp3/image_convert";
 const char *def_usr_img = 0;
 
 // image suffixes to search
diff -urNad vdr-plugin-mp3-0.10.2~/data.c vdr-plugin-mp3-0.10.2/data.c
--- vdr-plugin-mp3-0.10.2~/data.c	2009-11-29 12:42:14.000000000 +0100
+++ vdr-plugin-mp3-0.10.2/data.c	2010-01-23 20:36:23.000000000 +0100
@@ -39,7 +39,7 @@
 
 // ----------------------------------------------------------------
 
-const char *mountscript = "mount.sh";
+const char *mountscript = "/usr/share/vdr-plugin-mp3/mount";
 
 char *Quote(const char *str)
 {
diff -urNad vdr-plugin-mp3-0.10.2~/player-mplayer.c vdr-plugin-mp3-0.10.2/player-mplayer.c
--- vdr-plugin-mp3-0.10.2~/player-mplayer.c	2009-10-14 00:18:18.000000000 +0200
+++ vdr-plugin-mp3-0.10.2/player-mplayer.c	2010-01-23 20:36:23.000000000 +0100
@@ -47,7 +47,7 @@
 
 #define MPLAYER_VOL_STEP 3.0
 
-const char *MPlayerCmd = "mplayer.sh";
+const char *MPlayerCmd = "/usr/share/vdr-plugin-mplayer/mplayer";
 int MPlayerAid=-1;
 const char *globalResumeDir = 0;
 
